README.md

```diff
@@ -230,6 +230,12 @@ written:
   log will explain why the device is considered broken, and how to reinstate the
   device into the cluster after resolving the issue.
 
+* `/var/lib/swift-storage/broken` has the same structure and semantics as
+  `/run/swift-storage/broken`, but its contents are retained across reboots. A
+  flag from `/run/swift-storage/broken` can be copied to
```

Comment:
Copying might not be possible. In such cases we would modify the node label responsible for running swift related sw and just create `ln -s  /dev/<device> /var/lib/swift-storage/broken/<serial>`, correct?

Reply:
That is correct, but what makes you think that copying would not be possible?

Reply:
because the kernel panic might hit before someone can login and do the copy